hera_mc/geo_handling.py

```diff
@@ -238,8 +244,8 @@ def get_location(self, to_find, query_date, show_location=False, verbosity='m'):
                             this_station = 'No station type data.'
                     a.gps2Time()
                     desc = self.station_types[this_station]['Description']
-                    ever_connected = self.is_in_connections(a.station_name, '<')
-                    active = self.is_in_connections(a.station_name, query_date)
+                    ever_connected = self.is_in_connections(a.station_name)
+                    active = self.is_in_connections(a.station_name, query_date, True)
```

Comment:
Please write out `return_antrev=True` in the function call to clarify that kind of data is being returned.